README.md

```diff
@@ -1 +1,154 @@
-# kanban-board-setup
\ No newline at end of file
+<div id="top"></div>
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#jscapstone">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#authors">Authors</a></li>
+    <li><a href="#support">Support</a></li>
+    <li><a href="#license">License</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+# JavaScript Capstone Project
+
```

Comment:
- [ **OPTIONAL** ] What do you think if you give your project a meaningful name with a short idea about its content to make it more professional? e.g.: iCinema, MoviesInfos,... a thing similar to that